include/SupportedUserTypes.h

```diff
@@ -285,17 +285,31 @@ namespace ChimeraTK {
 
   template<typename NUMERIC>
   NUMERIC detail::userTypeToNumeric_impl<std::string, NUMERIC>::impl(const std::string& value) {
+    bool isHexidecimal = value.length() > 1 and (value[1] == 'x' or value[1] == 'X'); // OxABCD form hex
+
     if constexpr(std::is_same<NUMERIC, Boolean>::value) {
       // special treatment for Boolean
-      std::stringstream ss(value);
+      std::stringstream ss;
       Boolean converted;
+      if(isHexidecimal) {
+        ss << std::hex << value;
+      }
+      else {
+        ss.str(value);
+      }
       ss >> converted;
       return converted;
     }
```

Comment:
Actually, I am wondering why there is a special case for Boolean. It is (and was) the same code as for the not  (u)int_8. We should clean this up and delete the unnecessary code for Boolean (or am I overlooking something)?

Reply:
This special case seems to be safely removable. I tried removing it and was still able to parse into Boolean without a problem.